### PR TITLE
[IMP] pos_restaurant:test merging of table in edit mode

### PR DESCRIPTION
**Pull request by SalukiMakingCode**
Tour steps:
implement the test for mergin table in PoS on edit mode

merge the table in edit mode with link button
check that table are merge

tasks id : 3748795

### CODE REVIEW

**Review by vlst-odoo — CHANGES_REQUESTED** [review]
After [this](https://github.com/odoo/odoo/pull/152892) is merged, please insert the steps from this test into the existing one

File: addons/pos_restaurant/static/tests/tours/pos_restaurant.js [hunk 1]
```diff
@@ -292,5 +292,85 @@ registry.category("web_tour.tours").add("MergeTableTour", {
...
+            {
+                content: `Verify table 4 and 5 is merge`,
+                trigger: 'div.table div.label:contains("4")',
+                isCheck: true,
+                run: () => {
+                    if ($("div.table div.label:contains('4')").length < 2) {
+                        throw new TourError("Table isn't merge");
+                    }
+                },
+            },
```
Comment (vlst-odoo) [inline addons/pos_restaurant/static/tests/tours/pos_restaurant.js:374]: now that you put these steps in the existing tour you don't need this one, right?

Comment (SalukiMakingCode) [reply addons/pos_restaurant/static/tests/tours/pos_restaurant.js]: indeed :D

**Comment by vlst-odoo** [comment]
robodoo r+

**Pull request closed without merging**